lib/pure/encodings.nim

```diff
@@ -282,8 +282,10 @@ else:
 
   var errno {.importc, header: "<errno.h>".}: cint
 
-  when defined(freebsd) or defined(netbsd):
+  when defined(bsd):
     {.pragma: importIconv, cdecl, header: "<iconv.h>".}
+    when defined(openbsd):
+      {.passL: "-I/usr/local/include".}
```

Comment:
see whether it works, maybe OpenBSD should always link this path

Reply:
i dont' understand: passL is for linker, -I is for compilation phase

Reply:
well, what do you think how can I think I should use?
I want to link `/usr/local/include/libiconv.a`.